content/docs/getting-started-tutorial/building-your-data-model.md

```diff
@@ -0,0 +1,112 @@
+5. Check your connection is working:
+
+- Press `Test Connection` to check that the connection is working.
+- Once this is successful, you can press `Save Connection`.
+
+<img src="https://assets.dataform.co/getting%20started%20tutorial/creating%20a%20dataset/Screenshot%202020-08-13%20at%2015.46%201%20(1).png" max-width="753"  alt="Connecting a warehouse" />
+
+## Creating a dataset
+
+Now that you've created your project and connected a warehouse, you're ready to start defining your data model.
+
+1. Make sure you are in a development branch:
+
+- By default you should already be in a development branch called `yourname_dev`.
+- However, if you would like to create a new branch, click `Develop` and select `New Branch`.
+- Give your branch a name.
+
+2. Create a new dataset:
+
+- Click on the `New Dataset` button in the left hand side bar.
+- Choose whether you want your dataset to be a table, view or incremental table. In this case we want to create a table.
+- Name the table `order_stats` and click `Create Table`.
+
+<div className="bp3-callout bp3-icon-info-sign bp3-intent-primary" markdown="1">
+<h4 class="bp3-heading">In Dataform transformations are defined using SQLX</h4>
+ SQLX is an extension of SQL. A typical SQLX file contains a SELECT statement defining a new table or a view and a config block at the top. The config block is used to specify additional options for the table or view. To see a full list of options for the config block, open the Documentation tab.
+</a></div>
+
+3. Define the dataset:
+
+- To create your table you're going to be using the below query which joins orders information from Shopify, payment details from Stripe and also applies some filters to the data:
+
+```sql
+SELECT
+  orders.date AS order_date,
+  orders.id AS id,
+  orders.customer_id AS customer_id,
+  orders.status AS order_status,
+  charges.status AS payment_status,
+  charges.payment_method AS payment_method,
+  SUM(orders.item_count) AS item_count,
+  SUM(charges.amount) AS amount
+
+FROM
+  dataform-demos.dataform_tutorial.shopify_orders AS orders
+  LEFT JOIN dataform-demos.dataform_tutorial.stripe_payments AS charges
+    ON orders.payment_id = charges.id
+
+WHERE
+  orders.id <= 999
+  AND orders.item_count > 0
+  AND orders.status <> 'internal'
+  AND charges.payment_method IN ('debit_card', 'subscription', 'coupon')
+
+GROUP BY 1, 2, 3, 4, 5, 6
+```
+
+- Paste the query into your file below the config block.
+- Dataform will automatically validate your query and check for any errors
+- Once you see that the query is valid you can click `Preview Results` to check that the data looks correct
+
+4.  Create the table in your warehouse:
+
+- Click `Publish Table` to create the table in your warehouse
+- This will take the SQLX that we’ve written, compile it into pure SQL, and then execute that SQL in your warehouse with the correct boilerplate code to create a table
```

Comment:
```suggestion
- This will take the SQLX that we’ve written, compile it into the SQL syntax of your warehouse (in this case, BigQuery), and then execute that SQL in your warehouse with the correct boilerplate code to create a table
```